cmd.go

```diff
@@ -116,9 +117,9 @@ func (f ArgsFn) Set(args []string) error { return f(args) }
 // config is configuration for root command.
 type config struct {
 	subConfig
-	name          string
-	errorHandling flag.ErrorHandling
-	output        io.Writer
+	name              string
```

Comment:
Please format the file, there are redundant spaces here.

Reply:
👍